keel-core/src/main/kotlin/com/netflix/spinnaker/keel/artifacts/DebianArtifact.kt

```diff
@@ -16,9 +16,16 @@ data class DebianArtifact(
   override val reference: String = name,
   val vmOptions: VirtualMachineOptions,
   override val statuses: Set<ArtifactStatus> = emptySet(),
-  override val versioningStrategy: VersioningStrategy = DebianVersioningStrategy,
   override val from: ArtifactOriginFilterSpec? = null
 ) : DeliveryArtifact() {
   override val type = DEBIAN
+
+  override val sortingStrategy: SortingStrategy
```

Comment:
The sorting strategy is auto-selected based on whether the artifact spec is defined in terms of source details or not.